### PR TITLE
[GPV-801] Adds super-linter task and changes

### PR DESCRIPTION
**Pull request by degaurab**
- Add `make lint` target to run the linter using shellcheck
    - Add `make super-linter` target to run linter in docker
    - Only including bash scripts checking
    - Exclude the 00_compile_tpcds/tests/ folder: because that's directly
      included from the TPC-DS toolkit, and we should NOT change them
    - Fixed warnings and errors listed in shellcheck linter

Co-authored-by: Gaurab Dey [gaurabd@vmware.com](mailto:gaurabd@vmware.com)
Co-authored-by: Mike Nemesh [nemeshm@vmware.com](mailto:nemeshm@vmware.com)
Co-authored-by: Xin Zhang <zhxin@vmware.com>

### CODE REVIEW

File: 04_load/rollout.sh [hunk 1]
```diff
@@ -35,24 +36,24 @@ start_gpfdist()
...
-			CHILD=$(echo $i | awk -F '|' '{print $1}')
-			EXT_HOST=$(echo $i | awk -F '|' '{print $2}')
-			GEN_DATA_PATH=$(echo $i | awk -F '|' '{print $3}')
-			GEN_DATA_PATH=$GEN_DATA_PATH/dsbenchmark
-			PORT=$(($GPFDIST_PORT + $CHILD))
+			CHILD=$(echo "$i" | awk -F '|' '{print $1}')
+			EXT_HOST=$(echo "$i" | awk -F '|' '{print $2}')
+			GEN_DATA_PATH=$(echo "$i" | awk -F '|' '{print $3}')
+			GEN_DATA_PATH="$GEN_DATA_PATH"/dsbenchmark
+			PORT=$("$GPFDIST_PORT" + "$CHILD")
```
Comment (stuart-pollock) [inline 04_load/rollout.sh:54]: Is this correct?  I would have expected a calculation to look something like:
```
PORT=$((GPFDIST_PORT + CHILD))
```

Comment (degaurab) [reply 04_load/rollout.sh]: @stuart-pollock you are correct missed this one.
As per option I was leaning more toward

```
PORT=$(($GPFDIST_PORT + $CHILD))
```
add `$` on variable for clear separation for variable and function calls.

**Review by Chibin — APPROVED** [review]

**Comment by Chibin** [comment]
Verified with a 1G TPC-DS run.

**Review by xinzweb — APPROVED** [review]
Great!

**Review by Chibin — APPROVED** [review]

**Pull request merged**